packages/sign-client/src/client.ts

```diff
@@ -209,7 +214,12 @@ export class SignClient extends ISignClient {
       await this.session.init();
       await this.proposal.init();
       await this.pendingRequest.init();
-      await this.engine.init();
+
+      if (this.session.getAll().length > 0) {
```

Comment:
What does this mean?
nit: move into variable with a name to depict why you're checking this

Reply:
I'll move it to a var so its more clear. When `session` is initialized, it loads the current sessions in, using `restore()`. I am checking to see if there are any active sessions